packages/amplication-client/src/Entity/EntityFieldProperty.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Icon } from "@amplication/design-system";
+import "./EntityFieldProperty.scss";
+type Props = {
+  property: string;
+  icon: string;
+};
+
+const CLASS_NAME = "entity-field-list-item-property";
+
+export const EntityFieldProperty = ({ property, icon }: Props) => {
```

Comment:
instead this declaration change it to react best practice:
export const EntityFieldProperty: React.FC<Props> = ({property, icon}) => {

Reply:
Done✅